02-IDE_exercises/solutions/06c-project/std_dev.py

```diff
@@ -9,21 +9,21 @@ class Person:
     Represents a person
     """
 
-    def __init__(self, name, age):
+    def __init__(self, name, age) -> None:
```

Comment:
I'm not sure what best practice for `__init__` is... should it hint at `None`? I asked ChatGPT and it said it wasn't best practice for `__init__` to type hint, but that could be wrong. We should confirm what best practice is.